plugins/kubernetes/app/models/watchers/events/pod_event.rb

```diff
@@ -0,0 +1,26 @@
+module Watchers
+  module Events
+    class PodEvent
+      def initialize(event)
+        @event = event
+        @api_pod = Kubernetes::Api::Pod.new(event.object)
+      end
+
+      def ready?
```

Comment:
Maybe make @api_pod an ```attr_reader```, then consumers of this class can call ```pod_event.pod.ready?``` instead of having to create wrapping methods for each method you want to expose.